homework/PavelBo/collections/stringTokenizer/src/test/java/com/tutrit/java/quickstart/StringAnalyzerTest.java

```diff
@@ -0,0 +1,52 @@
+package com.tutrit.java.quickstart;
+
+import junit.framework.Assert;
+import junit.framework.TestCase;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Set;
+
+public class StringAnalyzerTest extends TestCase {
+
+    StringAnalyzer stringAnalyzer;
+
+    @Before
+    public void setUp() {
+        stringAnalyzer = new StringAnalyzer();
+    }
+
+    @Test
+    public void testSplitToSentences() {
+        List<String> sentences = stringAnalyzer.splitToSentences(Text.text);
+        Assert.assertEquals(17, sentences.size());
+    }
+
+    @Test
+    public void testSplitToWords (){
+        List<String> words = stringAnalyzer.splitToWords(Text.text);
```

Comment:
It's better not to use production data in unit tests, but simple data that obviously shows that method works. Replace Text.text with something simple